### ISSUE TITLE
Ignore failures option doesn't work for Gradle plugin

### ISSUE THREAD

**Issue by lukaville** [MarathonLabs/marathon#208]
There is an option in Gradle plugin extension "ignoreFailures":

```
marathon {
    ignoreFailures = true
}
```

But if you enabled it in the extension it will not be enabled because we have the following condition in the task:

```
ignoreFailure && (extensionConfig?.ignoreFailures ?: true)
```

Where ignoreFailure is a field of the task and it's `false` by default. What is the reason behind this? How to properly enable `ignoreFailures` mode for Gradle plugin?

**Comment by tagantroy** [comment]
Hello, I think it should be `extensionConfig.ignoreFailures ?: false`. I'll create PR

**Comment by tagantroy** [comment]
Fixed #211
